lib/event_store_client/store_adapter/in_memory.rb

```diff
@@ -46,7 +50,7 @@ def initialize(host:, port:, per_page: 20)
       def read_stream_backward(stream_name, start: 0, count: per_page)
         return [] unless event_store.key?(stream_name)
 
-        start = start.zero? ? event_store[stream_name].length - 1 : start
+        start = start == 'head' ? event_store[stream_name].length - 1 : start
```

Comment:
won't it crash if start will be a `0` ?

Reply:
0 is the first index of a stream.  `start: 0` in a backward request returns 1 event.